hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/constants/AbfsHttpConstants.java

```diff
@@ -76,6 +76,7 @@
   public static final String AT = "@";
   public static final String HTTP_HEADER_PREFIX = "x-ms-";
   public static final String HASH = "#";
+  public static final String TRUE = "true";
```

Comment:
This can be class level constant right, the lieral "true" as such does not have anuthing to do with HTTP related operations.

Reply:
Boolean query param value field. Will retain.